### ISSUE TITLE
Make the text appear bold/italic when \textbf \textit (and \emph) are applied?

### ISSUE THREAD

**Issue by linhtto** [viktorstrate/vscode-latex-utilities#4]
I hope that this is doable in VSCode the way it works in Sublime. But once a text is embedded within \textbf, several LaTeX packages in Sublime can make the font look bold within the editor, and similarly for \textit and \emph. If possible I think it would be a nice enhancement.

**Comment by viktorstrate** [comment]
This is possible, but the plugin [LaTeX-Workshop](https://marketplace.visualstudio.com/items?itemName=James-Yu.latex-workshop) already does this. Among a bunch of other things

Because this plugin is by far the biggest LaTeX plugin for vscode, latex-utilities is thought as an extension to that plugin, and "expects" the user to have that one already installed.

To prevent conflicts, and reinvention of the wheel.

**Comment by linhtto** [comment]
I did install that extension too but didn't see that feature at all in my
files. I think I even saw an old issue when someone suggested such a
feature but it didn't really get addressed. It does seem like something
suitable for this package if it doesn't exist already. Do you know how I
can turn it on?

**Comment by viktorstrate** [comment]
![image](https://user-images.githubusercontent.com/4233458/44838130-aec51680-ac3b-11e8-8b43-a58d8aa650ce.png)

Highlighting works out of the box for me, I have only `LaTeX-Workbshop v. 5.8.2` and `latex-utilities` installed.

And no user-defined settings relating syntax highlighting in LaTeX, using normal `.tex` files

**Comment by linhtto** [comment]
Thanks for checking. That's interesting. I also don't have any other syntax highlighting for latex.

So what I did is to try different color themes, and apparently that makes a difference, even though I only try within the default/pre-installed ones.

Of the defaults, only these work: 
- Light, Light Plus, Quiet Light
- Dark, Dark Plus, Kimbie Dark, and High Contrast

I was using Solarized Dark and it didn't work (but the rest of them not listed above don't work either). This is a VSCode issue then, because I used Solarized Dark with Sublime too and it was fine.
